packages/ui/hooks/fuse/useTVL.ts

```diff
@@ -12,14 +12,15 @@ export const fetchFuseNumberTVL = async (midasSdk: MidasSdk) => {
   return Number(utils.formatUnits(tvlNative, decimals));
 };
 
-interface CrossChainTVL {
-  [chainId: string]: {
+type CrossChainTVL = Map<
```

Comment:
You could also use `Record` if you like it better, it's then just a normal js object again. 🤷 
```typescript
Record<string,
{
     value: number;
     symbol: string;
     name: string;
     logo: string;
   }>
```

Reply:
I think Normal js object couldn't sort by values because it is sorted by keys automatically, so we should use Map. Am I missing something?